src/cluster/changeset/manager.go

```diff
@@ -130,7 +130,7 @@ func NewManager(opts ManagerOptions) (Manager, error) {
 
 	logger := opts.Logger()
 	if logger == nil {
-		logger = log.NullLogger
+		logger = zap.NewNop()
```

Comment:
They couldn't have named it `NewNoop` instead...

Reply:
heh, naming ..